conftest.py

```diff
@@ -214,6 +219,10 @@ def can_connect_to_mongo():
 
 @pytest.fixture(scope='function', params=['mongo', 'file', 'memory'])
```

Comment:
FWIW, I'd prefer an additional option that is the list of dbs to be tested, which then would provide the value for params.

Reply:
That's almost what I did. Would be easy enough to change if we feel it is worth it do to right now.

Reply:
I changed it to support this now. Call with, e.g. `pytest -xv --test-databases file memory`.  Supports an `all` option (which travis uses). Default 'mongo' only.